optimum/gptq/quantizer.py

```diff
@@ -77,6 +77,7 @@ def __init__(
         exllama_config: Dict[str, Any] = None,
         max_input_length: Optional[int] = None,
         cache_block_outputs: Optional[bool] = True,
+        inside_layer_modules: Optional[List[List[str]]] = None,
```

Comment:
Can we have a more explicit name like `modules_in_block_to_quantize`? Why is it a `List[List[str]]` and not list `List[str]`?

Reply:
related to the quantization process. See comment below